### ISSUE TITLE
@babel/runtime seems to be missing in dev dependencies 

### ISSUE THREAD

**Issue by lal12** [suren-atoyan/monaco-react#55]
@babel/runtime seems to be missing in dev dependencies

**Comment by ITAYC0HEN** [comment]
same here,
```
Module not found: Can't resolve '@babel/runtime/helpers/classCallCheck
```

**Comment by suren-atoyan** [comment]
Hello @lal12 @ITAYC0HEN.

This is fixed in version `3.5.5`.

Now `@babel/runtime` is a dependency.

Thank you for your support.
